### ISSUE TITLE
No Incoming Messages in Monitor

### ISSUE THREAD

**Issue by BeastBlood1885** [JocysCom/TextToSpeech#42]
The TTS Monitor doesn't seem to work for me.

Both the addon and the monitor are up-to-date.
The monitor is set to run as admin.
I have Display mode enabled. Nothing is obstructing the colored pixels at the top left, though I remember it actually working even with Titan Panel smack-dab in that position a few months back.
The Windows firewall isn't blocking anything. 
The "Speak" button and debug tests in the monitor work perfectly fine.

I see the colored squares change when new messages come up, but there's nothing in the "Incoming Messages" tab of the monitor.

**Comment by BeastBlood1885** [comment]
Update: Tried installing it on my laptop and it worked for about an hour before abruptly stopping.

**Comment by BeastBlood1885** [comment]
I figured it out!

The problem on my laptop was that, for some reason, there are two lines of colored pixels and one of them was hidden underneath the other. It worked when I brought up the Options menu in-game and set the TOP position to 10, so one chills at the top of the screen while the other sits just a bit below. 

For my desktop, I ran the /console cvar_default command and it restarted working. Some other addon must've messed with it.
